pytential/qbx/target_specific/impl.pyx

```diff
@@ -106,8 +106,8 @@ def h3dall_wrapper(nterms, z, scale, hs, hders):
         int nterms_, ifder
         double scale_
         double complex z_
-        double complex[:] hvec = np.empty(1 + nterms, np.complex)
-        double complex[:] hdervec = np.empty(1 + nterms, np.complex)
+        double complex[:] hvec = np.empty(1 + nterms, np.complex128)
```

Comment:
Not a cython person, so hopefully these don't have a change in behavior?

Reply:
Don't think so. If anything, I'd say this is an improvements. Plus the tests seem to pass.